apps/ui/src/composables/useSettings.ts

```diff
@@ -1,6 +1,67 @@
+import { clone } from '@/helpers/utils';
 import { getNetwork, metadataNetwork } from '@/networks';
 
-const settings = ref<Map<string, any>>(new Map());
+type LIMITS =
+  | 'space.active_proposal_limit_per_author'
+  | 'space.default.body_limit'
+  | 'space.default.choices_limit'
+  | 'space.default.proposal_limit_per_day'
+  | 'space.default.proposal_limit_per_month'
+  | 'space.default.strategies_limit'
+  | 'space.ecosystem.proposal_limit_per_day'
+  | 'space.ecosystem.proposal_limit_per_month'
+  | 'space.flagged.proposal_limit_per_day'
+  | 'space.flagged.proposal_limit_per_month'
+  | 'space.turbo.body_limit'
+  | 'space.turbo.choices_limit'
+  | 'space.turbo.proposal_limit_per_day'
+  | 'space.turbo.proposal_limit_per_month'
+  | 'space.turbo.strategies_limit'
+  | 'space.verified.body_limit'
+  | 'space.verified.choices_limit'
+  | 'space.verified.proposal_limit_per_day'
+  | 'space.verified.proposal_limit_per_month'
+  | 'space.verified.strategies_limit'
+  | 'user.default.follow_limit';
+
+type LISTS = 'space.ecosystem.list';
+
+const DEFAULT_LIMITS_SETTINGS: Record<LIMITS, number> = {
+  'space.active_proposal_limit_per_author': 0,
+  'space.default.body_limit': 0,
+  'space.default.choices_limit': 0,
+  'space.default.proposal_limit_per_day': 0,
+  'space.default.proposal_limit_per_month': 0,
+  'space.default.strategies_limit': 0,
+  'space.ecosystem.proposal_limit_per_day': 0,
+  'space.ecosystem.proposal_limit_per_month': 0,
+  'space.flagged.proposal_limit_per_day': 0,
+  'space.flagged.proposal_limit_per_month': 0,
+  'space.turbo.body_limit': 0,
+  'space.turbo.choices_limit': 0,
+  'space.turbo.proposal_limit_per_day': 0,
+  'space.turbo.proposal_limit_per_month': 0,
+  'space.turbo.strategies_limit': 0,
+  'space.verified.body_limit': 0,
+  'space.verified.choices_limit': 0,
+  'space.verified.proposal_limit_per_day': 0,
+  'space.verified.proposal_limit_per_month': 0,
+  'space.verified.strategies_limit': 0,
+  'user.default.follow_limit': 0
+};
+
+const DEFAULT_LISTS_SETTINGS: Record<LISTS, string[]> = {
+  'space.ecosystem.list': []
+};
+
+const limits = ref({
+  ...clone(DEFAULT_LIMITS_SETTINGS)
+});
+
+const lists = ref({
+  ...clone(DEFAULT_LISTS_SETTINGS)
+});
```

Comment:
```suggestion
const limits = ref({
  ...clone(DEFAULT_LIMITS_SETTINGS)
});

const lists = ref({
  ...clone(DEFAULT_LISTS_SETTINGS)
});
```

Do we need to spread those? Isn't clone enough?

+ nit. I would group those refs together (without blank line between those).